packages/collector/Collector_Helpers.php

```diff
@@ -0,0 +1,23 @@
+<?php
+function collector_get_tmpfile($name, $type)
+{
+    return $name . '-package-'. date('Y-m-d_H-i-s') . '.' . $type;
+}
+
+function collector_get_fakepass()
+{
+    $fakepass = base64_encode(random_bytes(128));
+
+    set_transient('COLLECTOR_FAKE_PASSWORD', $fakepass, 60 * 5);
```

Comment:
What's the value of storing it and passing as transient vs hardcoding it as `"password"`?

Reply:
Not sure, removing this also